2018-10-17 16:21:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Data conversion error converting "'A1' (ITEM: ITEM_NUMBER INT)"; SQL statement:
INSERT INTO ITEM (ITEM_NUMBER, NAME, DESCRIPTION, AVAILABLE_QUANTITY, UNIT_PRICE) 
VALUES ('A1', 'CPP','C PLUS PLUS',100,80.0) -- ('A1', 'CPP', 'C PLUS PLUS', 100, 80.0) [22018-197]
2018-10-17 16:25:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column count does not match; SQL statement:
INSERT INTO ITEM (ITEM_NUMBER, NAME, DESCRIPTION, AVAILABLE_QUANTITY, UNIT_PRICE) 
select * from ITEM [21002-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.command.dml.Insert.prepare(Insert.java:326)
	at org.h2.command.Parser.prepareCommand(Parser.java:283)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
2018-10-17 18:39:02 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:189)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:352)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl$6.run(WSManagedConnectionFactoryImpl.java:871)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl$6.run(WSManagedConnectionFactoryImpl.java:845)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.getConnectionUsingDS(WSManagedConnectionFactoryImpl.java:845)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.getConnection(WSManagedConnectionFactoryImpl.java:806)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.createManagedConnection(WSManagedConnectionFactoryImpl.java:656)
	at com.ibm.ejs.j2c.FreePool.createManagedConnectionWithMCWrapper(FreePool.java:1354)
	at com.ibm.ejs.j2c.FreePool.createOrWaitForConnection(FreePool.java:1228)
	at com.ibm.ejs.j2c.PoolManager.reserve(PoolManager.java:1434)
	at com.ibm.ejs.j2c.ConnectionManager.allocateMCWrapper(ConnectionManager.java:573)
	at com.ibm.ejs.j2c.ConnectionManager.allocateConnection(ConnectionManager.java:309)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:143)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:116)
	at org.eclipse.persistence.sessions.JNDIConnector.connect(JNDIConnector.java:135)
	at org.eclipse.persistence.sessions.DatasourceLogin.connectToDatasource(DatasourceLogin.java:170)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connectInternal(DatasourceAccessor.java:346)
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.connectInternal(DatabaseAccessor.java:313)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connect(DatasourceAccessor.java:434)
	at org.eclipse.persistence.sessions.server.ConnectionPool.buildConnection(ConnectionPool.java:217)
	at org.eclipse.persistence.sessions.server.ExternalConnectionPool.startUp(ExternalConnectionPool.java:146)
	at org.eclipse.persistence.sessions.server.ServerSession.connect(ServerSession.java:519)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.login(DatabaseSessionImpl.java:819)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.login(EntityManagerFactoryProvider.java:256)
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:761)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:216)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.createEntityManagerImpl(EntityManagerFactoryDelegate.java:324)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManagerImpl(EntityManagerFactoryImpl.java:348)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManager(EntityManagerFactoryImpl.java:329)
	at com.ibm.ws.jpa.container.v22.internal.JPA22Runtime.createEntityManagerInstance(JPA22Runtime.java:110)
	at com.ibm.ws.jpa.management.JPAEMPool.getEntityManager(JPAEMPool.java:155)
	at com.ibm.ws.jpa.management.JPATxEntityManager.getEMInvocationInfo(JPATxEntityManager.java:254)
	at com.ibm.ws.jpa.management.JPATxEntityManager.getEMInvocationInfo(JPATxEntityManager.java:164)
	at com.ibm.ws.jpa.management.JPAEntityManager.createQuery(JPAEntityManager.java:594)
	at edu.osu.cse5234.business.InventoryServiceBean.getAvailableInventory(InventoryServiceBean.java:33)
	at edu.osu.cse5234.business.view.EJSRemote0SLInventoryServiceBean_78e4cf4b.getAvailableInventory(EJSRemote0SLInventoryServiceBean_78e4cf4b.java)
	at edu.osu.cse5234.business.view._EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie._get_availableInventory(_EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie.java:1)
	at edu.osu.cse5234.business.view._EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie._invoke(_EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie.java)
	at org.apache.yoko.orb.OBPortableServer.ServantDispatcher.dispatch(ServantDispatcher.java:187)
	at org.apache.yoko.orb.OBPortableServer.POA_impl._OB_dispatch(POA_impl.java:1644)
	at org.apache.yoko.orb.OB.DispatchRequest_impl.invoke(DispatchRequest_impl.java:56)
	at com.ibm.ws.transport.iiop.yoko.ExecutorDispatchStrategy$1.run(ExecutorDispatchStrategy.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 68 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 62 more
2018-10-17 19:14:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "NEW" not found; SQL statement:
create table CUSTOMER_ORDER (ID int auto_increment primary key, CUSTOMER_NAME varchar(255), CUSTOMER_EMAIL varchar(255), SHIPPING_INFO_ID_FK int, PAYMENT_INFO_ID_FK int, STATUS varchar(255) default New) [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.table.Column.setDefaultExpression(Column.java:243)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4246)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:6662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6570)
	at org.h2.command.Parser.parseCreate(Parser.java:4615)
	at org.h2.command.Parser.parsePrepared(Parser.java:380)
	at org.h2.command.Parser.parse(Parser.java:335)
	at org.h2.command.Parser.parse(Parser.java:307)
	at org.h2.command.Parser.prepareCommand(Parser.java:278)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
2018-10-17 19:15:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "ËNEWÍ" not found; SQL statement:
create table CUSTOMER_ORDER (ID int auto_increment primary key, CUSTOMER_NAME varchar(255), CUSTOMER_EMAIL varchar(255), SHIPPING_INFO_ID_FK int, PAYMENT_INFO_ID_FK int, STATUS varchar(255) default ëNewí) [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.table.Column.setDefaultExpression(Column.java:243)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4246)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:6662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6570)
	at org.h2.command.Parser.parseCreate(Parser.java:4615)
	at org.h2.command.Parser.parsePrepared(Parser.java:380)
	at org.h2.command.Parser.parse(Parser.java:335)
	at org.h2.command.Parser.parse(Parser.java:307)
	at org.h2.command.Parser.prepareCommand(Parser.java:278)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
2018-10-17 19:15:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "CUSTOMER_ORDER_LINE_ITEM" already exists; SQL statement:
create table CUSTOMER_ORDER_LINE_ITEM (ID int auto_increment primary key, ITEM_ID int, ITEM_NAME varchar(255), QUANTITY int, CUSTOMER_ORDER_ID_FK int) [42101-197]
2018-10-17 19:15:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "PAYMENT_INFO" already exists; SQL statement:
create table PAYMENT_INFO (ID int auto_increment primary key, AMOUNT double) [42101-197]
2018-10-17 19:15:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "SHIPPING_INFO" already exists; SQL statement:
create table SHIPPING_INFO (ID int auto_increment primary key, AMOUNT double) [42101-197]
2018-10-17 19:15:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "ËNEWÍ" not found; SQL statement:
create table CUSTOMER_ORDER (ID int auto_increment primary key, CUSTOMER_NAME varchar(255), CUSTOMER_EMAIL varchar(255), SHIPPING_INFO_ID_FK int, PAYMENT_INFO_ID_FK int, STATUS varchar(255) default ëNewí) [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.table.Column.setDefaultExpression(Column.java:243)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4246)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:6662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6570)
	at org.h2.command.Parser.parseCreate(Parser.java:4615)
	at org.h2.command.Parser.parsePrepared(Parser.java:380)
	at org.h2.command.Parser.parse(Parser.java:335)
	at org.h2.command.Parser.parse(Parser.java:307)
	at org.h2.command.Parser.prepareCommand(Parser.java:278)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
2018-10-17 19:15:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "NEW" not found; SQL statement:
create table CUSTOMER_ORDER (ID int auto_increment primary key, CUSTOMER_NAME varchar(255), CUSTOMER_EMAIL varchar(255), SHIPPING_INFO_ID_FK int, PAYMENT_INFO_ID_FK int, STATUS varchar(255) default New) [42122-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:150)
	at org.h2.table.Column.setDefaultExpression(Column.java:243)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:4246)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:6662)
	at org.h2.command.Parser.parseCreateTable(Parser.java:6570)
	at org.h2.command.Parser.parseCreate(Parser.java:4615)
	at org.h2.command.Parser.parsePrepared(Parser.java:380)
	at org.h2.command.Parser.parse(Parser.java:335)
	at org.h2.command.Parser.parse(Parser.java:307)
	at org.h2.command.Parser.prepareCommand(Parser.java:278)
	at org.h2.engine.Session.prepareLocal(Session.java:611)
	at org.h2.engine.Session.prepareCommand(Session.java:549)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1247)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1077)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1386)
	at org.h2.server.web.WebApp.query(WebApp.java:1060)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1022)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1009)
	at org.h2.server.web.WebThread.process(WebThread.java:165)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
2018-10-22 13:54:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DROP TABLE PAYMENT_INFO
DROP[*] TABLE SHIPPING_INFO"; SQL statement:
DROP TABLE PAYMENT_INFO
DROP TABLE SHIPPING_INFO [42000-197]
2018-10-22 13:54:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DROP TABLE PAYMENT_INFO
DROP[*] TABLE SHIPPING_INFO"; SQL statement:
DROP TABLE PAYMENT_INFO
DROP TABLE SHIPPING_INFO [42000-197]
2018-10-22 13:54:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DROP TABLE PAYMENT_INFO
DROP[*] TABLE SHIPPING_INFO"; SQL statement:
DROP TABLE PAYMENT_INFO
DROP TABLE SHIPPING_INFO [42000-197]
2018-10-22 13:54:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DROP TABLE PAYMENT_INFO
DROP[*] TABLE SHIPPING_INFO"; SQL statement:
DROP TABLE PAYMENT_INFO
DROP TABLE SHIPPING_INFO [42000-197]
2018-10-22 13:54:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DROP TABLE PAYMENT_INFO
DROP[*] TABLE SHIPPING_INFO"; SQL statement:
DROP TABLE PAYMENT_INFO
DROP TABLE SHIPPING_INFO [42000-197]
2018-10-22 18:06:02 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Logan/Documents/workspace/cse5234/proj_5234/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:189)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:352)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl$6.run(WSManagedConnectionFactoryImpl.java:871)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl$6.run(WSManagedConnectionFactoryImpl.java:845)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.getConnectionUsingDS(WSManagedConnectionFactoryImpl.java:845)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.getConnection(WSManagedConnectionFactoryImpl.java:806)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.createManagedConnection(WSManagedConnectionFactoryImpl.java:656)
	at com.ibm.ejs.j2c.FreePool.createManagedConnectionWithMCWrapper(FreePool.java:1354)
	at com.ibm.ejs.j2c.FreePool.createOrWaitForConnection(FreePool.java:1228)
	at com.ibm.ejs.j2c.PoolManager.reserve(PoolManager.java:1434)
	at com.ibm.ejs.j2c.ConnectionManager.allocateMCWrapper(ConnectionManager.java:573)
	at com.ibm.ejs.j2c.ConnectionManager.allocateConnection(ConnectionManager.java:309)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:143)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:116)
	at org.eclipse.persistence.sessions.JNDIConnector.connect(JNDIConnector.java:135)
	at org.eclipse.persistence.sessions.DatasourceLogin.connectToDatasource(DatasourceLogin.java:162)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connectInternal(DatasourceAccessor.java:346)
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.connectInternal(DatabaseAccessor.java:309)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connect(DatasourceAccessor.java:434)
	at org.eclipse.persistence.sessions.server.ConnectionPool.buildConnection(ConnectionPool.java:217)
	at org.eclipse.persistence.sessions.server.ExternalConnectionPool.startUp(ExternalConnectionPool.java:146)
	at org.eclipse.persistence.sessions.server.ServerSession.connect(ServerSession.java:504)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.login(DatabaseSessionImpl.java:778)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.login(EntityManagerFactoryProvider.java:255)
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:757)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:207)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.createEntityManagerImpl(EntityManagerFactoryDelegate.java:307)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManagerImpl(EntityManagerFactoryImpl.java:337)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManager(EntityManagerFactoryImpl.java:318)
	at com.ibm.ws.jpa.container.v21.internal.JPA21Runtime.createEntityManagerInstance(JPA21Runtime.java:110)
	at com.ibm.ws.jpa.management.JPAEMPool.getEntityManager(JPAEMPool.java:155)
	at com.ibm.ws.jpa.management.JPATxEntityManager.getEMInvocationInfo(JPATxEntityManager.java:254)
	at com.ibm.ws.jpa.management.JPATxEntityManager.getEMInvocationInfo(JPATxEntityManager.java:164)
	at com.ibm.ws.jpa.management.JPAEntityManager.createQuery(JPAEntityManager.java:594)
	at edu.osu.cse5234.business.InventoryServiceBean.getAvailableInventory(InventoryServiceBean.java:33)
	at edu.osu.cse5234.business.view.EJSRemote0SLInventoryServiceBean_78e4cf4b.getAvailableInventory(EJSRemote0SLInventoryServiceBean_78e4cf4b.java)
	at edu.osu.cse5234.business.view._EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie._get_availableInventory(_EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie.java:1)
	at edu.osu.cse5234.business.view._EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie._invoke(_EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie.java)
	at org.apache.yoko.orb.OBPortableServer.ServantDispatcher.dispatch(ServantDispatcher.java:187)
	at org.apache.yoko.orb.OBPortableServer.POA_impl._OB_dispatch(POA_impl.java:1644)
	at org.apache.yoko.orb.OB.DispatchRequest_impl.invoke(DispatchRequest_impl.java:56)
	at com.ibm.ws.transport.iiop.yoko.ExecutorDispatchStrategy$1.run(ExecutorDispatchStrategy.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Logan/Documents/workspace/cse5234/proj_5234/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 68 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/Logan/Documents/workspace/cse5234/proj_5234/h2db/MyBookDB.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 62 more
2018-10-22 18:10:14 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Logan/Documents/workspace/cse5234/proj_5234/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:189)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:352)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl$6.run(WSManagedConnectionFactoryImpl.java:871)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl$6.run(WSManagedConnectionFactoryImpl.java:845)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.getConnectionUsingDS(WSManagedConnectionFactoryImpl.java:845)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.getConnection(WSManagedConnectionFactoryImpl.java:806)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.createManagedConnection(WSManagedConnectionFactoryImpl.java:656)
	at com.ibm.ejs.j2c.FreePool.createManagedConnectionWithMCWrapper(FreePool.java:1354)
	at com.ibm.ejs.j2c.FreePool.createOrWaitForConnection(FreePool.java:1228)
	at com.ibm.ejs.j2c.PoolManager.reserve(PoolManager.java:1434)
	at com.ibm.ejs.j2c.ConnectionManager.allocateMCWrapper(ConnectionManager.java:573)
	at com.ibm.ejs.j2c.ConnectionManager.allocateConnection(ConnectionManager.java:309)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:143)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:116)
	at org.eclipse.persistence.sessions.JNDIConnector.connect(JNDIConnector.java:135)
	at org.eclipse.persistence.sessions.DatasourceLogin.connectToDatasource(DatasourceLogin.java:162)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connectInternal(DatasourceAccessor.java:346)
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.connectInternal(DatabaseAccessor.java:309)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connect(DatasourceAccessor.java:434)
	at org.eclipse.persistence.sessions.server.ConnectionPool.buildConnection(ConnectionPool.java:217)
	at org.eclipse.persistence.sessions.server.ExternalConnectionPool.startUp(ExternalConnectionPool.java:146)
	at org.eclipse.persistence.sessions.server.ServerSession.connect(ServerSession.java:504)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.login(DatabaseSessionImpl.java:778)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.login(EntityManagerFactoryProvider.java:255)
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:757)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:207)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.createEntityManagerImpl(EntityManagerFactoryDelegate.java:307)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManagerImpl(EntityManagerFactoryImpl.java:337)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManager(EntityManagerFactoryImpl.java:318)
	at com.ibm.ws.jpa.container.v21.internal.JPA21Runtime.createEntityManagerInstance(JPA21Runtime.java:110)
	at com.ibm.ws.jpa.management.JPAEMPool.getEntityManager(JPAEMPool.java:155)
	at com.ibm.ws.jpa.management.JPATxEntityManager.getEMInvocationInfo(JPATxEntityManager.java:254)
	at com.ibm.ws.jpa.management.JPATxEntityManager.getEMInvocationInfo(JPATxEntityManager.java:164)
	at com.ibm.ws.jpa.management.JPAEntityManager.createQuery(JPAEntityManager.java:594)
	at edu.osu.cse5234.business.InventoryServiceBean.getAvailableInventory(InventoryServiceBean.java:33)
	at edu.osu.cse5234.business.view.EJSRemote0SLInventoryServiceBean_78e4cf4b.getAvailableInventory(EJSRemote0SLInventoryServiceBean_78e4cf4b.java)
	at edu.osu.cse5234.business.view._EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie._get_availableInventory(_EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie.java:1)
	at edu.osu.cse5234.business.view._EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie._invoke(_EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie.java)
	at org.apache.yoko.orb.OBPortableServer.ServantDispatcher.dispatch(ServantDispatcher.java:187)
	at org.apache.yoko.orb.OBPortableServer.POA_impl._OB_dispatch(POA_impl.java:1644)
	at org.apache.yoko.orb.OB.DispatchRequest_impl.invoke(DispatchRequest_impl.java:56)
	at com.ibm.ws.transport.iiop.yoko.ExecutorDispatchStrategy$1.run(ExecutorDispatchStrategy.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/Logan/Documents/workspace/cse5234/proj_5234/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 68 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/Logan/Documents/workspace/cse5234/proj_5234/h2db/MyBookDB.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 62 more
2018-10-23 11:49:37 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:189)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:352)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl$6.run(WSManagedConnectionFactoryImpl.java:871)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl$6.run(WSManagedConnectionFactoryImpl.java:845)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.getConnectionUsingDS(WSManagedConnectionFactoryImpl.java:845)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.getConnection(WSManagedConnectionFactoryImpl.java:806)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.createManagedConnection(WSManagedConnectionFactoryImpl.java:656)
	at com.ibm.ejs.j2c.FreePool.createManagedConnectionWithMCWrapper(FreePool.java:1354)
	at com.ibm.ejs.j2c.FreePool.createOrWaitForConnection(FreePool.java:1228)
	at com.ibm.ejs.j2c.PoolManager.reserve(PoolManager.java:1434)
	at com.ibm.ejs.j2c.ConnectionManager.allocateMCWrapper(ConnectionManager.java:573)
	at com.ibm.ejs.j2c.ConnectionManager.allocateConnection(ConnectionManager.java:309)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:143)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:116)
	at org.eclipse.persistence.sessions.JNDIConnector.connect(JNDIConnector.java:135)
	at org.eclipse.persistence.sessions.DatasourceLogin.connectToDatasource(DatasourceLogin.java:170)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connectInternal(DatasourceAccessor.java:346)
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.connectInternal(DatabaseAccessor.java:313)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connect(DatasourceAccessor.java:434)
	at org.eclipse.persistence.sessions.server.ConnectionPool.buildConnection(ConnectionPool.java:217)
	at org.eclipse.persistence.sessions.server.ExternalConnectionPool.startUp(ExternalConnectionPool.java:146)
	at org.eclipse.persistence.sessions.server.ServerSession.connect(ServerSession.java:519)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.login(DatabaseSessionImpl.java:819)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.login(EntityManagerFactoryProvider.java:256)
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:761)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:216)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.createEntityManagerImpl(EntityManagerFactoryDelegate.java:324)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManagerImpl(EntityManagerFactoryImpl.java:348)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManager(EntityManagerFactoryImpl.java:329)
	at com.ibm.ws.jpa.container.v22.internal.JPA22Runtime.createEntityManagerInstance(JPA22Runtime.java:110)
	at com.ibm.ws.jpa.management.JPAEMPool.getEntityManager(JPAEMPool.java:155)
	at com.ibm.ws.jpa.management.JPATxEntityManager.getEMInvocationInfo(JPATxEntityManager.java:254)
	at com.ibm.ws.jpa.management.JPATxEntityManager.getEMInvocationInfo(JPATxEntityManager.java:164)
	at com.ibm.ws.jpa.management.JPAEntityManager.createQuery(JPAEntityManager.java:594)
	at edu.osu.cse5234.business.InventoryServiceBean.getAvailableInventory(InventoryServiceBean.java:33)
	at edu.osu.cse5234.business.view.EJSRemote0SLInventoryServiceBean_78e4cf4b.getAvailableInventory(EJSRemote0SLInventoryServiceBean_78e4cf4b.java)
	at edu.osu.cse5234.business.view._EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie._get_availableInventory(_EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie.java:1)
	at edu.osu.cse5234.business.view._EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie._invoke(_EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie.java)
	at org.apache.yoko.orb.OBPortableServer.ServantDispatcher.dispatch(ServantDispatcher.java:187)
	at org.apache.yoko.orb.OBPortableServer.POA_impl._OB_dispatch(POA_impl.java:1644)
	at org.apache.yoko.orb.OB.DispatchRequest_impl.invoke(DispatchRequest_impl.java:56)
	at com.ibm.ws.transport.iiop.yoko.ExecutorDispatchStrategy$1.run(ExecutorDispatchStrategy.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 68 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 62 more
2018-10-23 16:44:02 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:733)
	at org.h2.server.web.WebApp.login(WebApp.java:954)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:134)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 29 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 23 more
2018-10-23 16:52:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DELETE *[*] FROM CUSTOMER_ORDER_LINE_ITEM "; expected "identifier"; SQL statement:
delete * from CUSTOMER_ORDER_LINE_ITEM [42001-197]
2018-10-23 16:52:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DELETE *[*] FROM CUSTOMER_ORDER_LINE_ITEM "; expected "identifier"; SQL statement:
delete * from CUSTOMER_ORDER_LINE_ITEM [42001-197]
2018-10-23 16:52:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DELETE *[*] FROM CUSTOMER_ORDER_LINE_ITEM "; expected "identifier"; SQL statement:
DELETE * from CUSTOMER_ORDER_LINE_ITEM [42001-197]
2018-10-23 17:12:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Value too long for column "CVV VARCHAR(3)": "'2341' (4)"; SQL statement:
INSERT INTO PAYMENT_INFO (HOLDER_NAME, CARD_NUM, EXP_DATE, CVV) VALUES (?, ?, ?, ?) [22001-197]
2018-10-23 17:13:04 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:733)
	at org.h2.server.web.WebApp.login(WebApp.java:954)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:134)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 29 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/weiyao/Documents/code/5234/lab6/h2db/MyBookDB.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 23 more
2018-10-24 15:52:44 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/derenkong/Desktop/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:733)
	at org.h2.server.web.WebApp.login(WebApp.java:954)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:134)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/derenkong/Desktop/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 29 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/derenkong/Desktop/lab6/h2db/MyBookDB.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 23 more
2018-10-24 15:53:49 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/derenkong/Desktop/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:733)
	at org.h2.server.web.WebApp.login(WebApp.java:954)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:134)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/derenkong/Desktop/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 29 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/derenkong/Desktop/lab6/h2db/MyBookDB.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 23 more
2018-10-24 15:54:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "ID[*]"; expected "INSERT, {"; SQL statement:
ID [42001-197]
2018-10-24 16:01:43 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/derenkong/Desktop/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:307)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:95)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:2506)
	at org.h2.mvstore.MVStore.panic(MVStore.java:383)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:364)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2934)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:155)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:100)
	at org.h2.engine.Database.getPageStore(Database.java:2538)
	at org.h2.engine.Database.open(Database.java:709)
	at org.h2.engine.Database.openDatabase(Database.java:286)
	at org.h2.engine.Database.<init>(Database.java:280)
	at org.h2.engine.Engine.openSession(Engine.java:66)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.jdbcx.JdbcDataSource.getJdbcConnection(JdbcDataSource.java:189)
	at org.h2.jdbcx.JdbcDataSource.getXAConnection(JdbcDataSource.java:352)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl$7.run(WSManagedConnectionFactoryImpl.java:1009)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl$7.run(WSManagedConnectionFactoryImpl.java:983)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.getConnectionUsingDS(WSManagedConnectionFactoryImpl.java:983)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.getConnection(WSManagedConnectionFactoryImpl.java:814)
	at com.ibm.ws.rsadapter.impl.WSManagedConnectionFactoryImpl.createManagedConnection(WSManagedConnectionFactoryImpl.java:664)
	at com.ibm.ejs.j2c.FreePool.createManagedConnectionWithMCWrapper(FreePool.java:1357)
	at com.ibm.ejs.j2c.FreePool.createOrWaitForConnection(FreePool.java:1231)
	at com.ibm.ejs.j2c.PoolManager.reserve(PoolManager.java:1448)
	at com.ibm.ejs.j2c.ConnectionManager.allocateMCWrapper(ConnectionManager.java:573)
	at com.ibm.ejs.j2c.ConnectionManager.allocateConnection(ConnectionManager.java:309)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:143)
	at com.ibm.ws.rsadapter.jdbc.WSJdbcDataSource.getConnection(WSJdbcDataSource.java:116)
	at org.eclipse.persistence.sessions.JNDIConnector.connect(JNDIConnector.java:135)
	at org.eclipse.persistence.sessions.DatasourceLogin.connectToDatasource(DatasourceLogin.java:162)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connectInternal(DatasourceAccessor.java:346)
	at org.eclipse.persistence.internal.databaseaccess.DatabaseAccessor.connectInternal(DatabaseAccessor.java:309)
	at org.eclipse.persistence.internal.databaseaccess.DatasourceAccessor.connect(DatasourceAccessor.java:434)
	at org.eclipse.persistence.sessions.server.ConnectionPool.buildConnection(ConnectionPool.java:217)
	at org.eclipse.persistence.sessions.server.ExternalConnectionPool.startUp(ExternalConnectionPool.java:146)
	at org.eclipse.persistence.sessions.server.ServerSession.connect(ServerSession.java:504)
	at org.eclipse.persistence.internal.sessions.DatabaseSessionImpl.login(DatabaseSessionImpl.java:786)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryProvider.login(EntityManagerFactoryProvider.java:258)
	at org.eclipse.persistence.internal.jpa.EntityManagerSetupImpl.deploy(EntityManagerSetupImpl.java:757)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.getAbstractSession(EntityManagerFactoryDelegate.java:207)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryDelegate.createEntityManagerImpl(EntityManagerFactoryDelegate.java:307)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManagerImpl(EntityManagerFactoryImpl.java:337)
	at org.eclipse.persistence.internal.jpa.EntityManagerFactoryImpl.createEntityManager(EntityManagerFactoryImpl.java:318)
	at com.ibm.ws.jpa.container.v21.internal.JPA21Runtime.createEntityManagerInstance(JPA21Runtime.java:110)
	at com.ibm.ws.jpa.management.JPAEMPool.getEntityManager(JPAEMPool.java:155)
	at com.ibm.ws.jpa.management.JPATxEntityManager.getEMInvocationInfo(JPATxEntityManager.java:254)
	at com.ibm.ws.jpa.management.JPATxEntityManager.getEMInvocationInfo(JPATxEntityManager.java:164)
	at com.ibm.ws.jpa.management.JPAEntityManager.createQuery(JPAEntityManager.java:594)
	at edu.osu.cse5234.business.InventoryServiceBean.getAvailableInventory(InventoryServiceBean.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.ibm.ejs.container.EJSContainer.invokeProceed(EJSContainer.java:4886)
	at com.ibm.ejs.container.interceptors.InvocationContextImpl.proceed(InvocationContextImpl.java:592)
	at com.ibm.ws.cdi.ejb.impl.InterceptorChain.proceed(InterceptorChain.java:119)
	at com.ibm.ws.cdi.ejb.impl.EJBCDIInterceptorWrapper.invokeInterceptors(EJBCDIInterceptorWrapper.java:131)
	at com.ibm.ws.cdi.ejb.impl.EJBCDIInterceptorWrapper.aroundInvoke(EJBCDIInterceptorWrapper.java:54)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.ibm.ejs.container.interceptors.InterceptorProxy.invokeInterceptor(InterceptorProxy.java:200)
	at com.ibm.ejs.container.interceptors.InvocationContextImpl.proceed(InvocationContextImpl.java:577)
	at org.jboss.weld.ejb.AbstractEJBRequestScopeActivationInterceptor.aroundInvoke(AbstractEJBRequestScopeActivationInterceptor.java:73)
	at com.ibm.ws.cdi.ejb.interceptor.WeldSessionBeanInterceptorWrapper.aroundInvoke(WeldSessionBeanInterceptorWrapper.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.ibm.ejs.container.interceptors.InterceptorProxy.invokeInterceptor(InterceptorProxy.java:200)
	at com.ibm.ejs.container.interceptors.InvocationContextImpl.proceed(InvocationContextImpl.java:577)
	at com.ibm.ejs.container.interceptors.InvocationContextImpl.doAroundInterceptor(InvocationContextImpl.java:298)
	at com.ibm.ejs.container.interceptors.InvocationContextImpl.doAroundInvoke(InvocationContextImpl.java:267)
	at com.ibm.ejs.container.EJSContainer.invoke(EJSContainer.java:4788)
	at edu.osu.cse5234.business.view.EJSRemote0SLInventoryServiceBean_78e4cf4b.getAvailableInventory(EJSRemote0SLInventoryServiceBean_78e4cf4b.java)
	at edu.osu.cse5234.business.view._EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie._get_availableInventory(_EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie.java:1)
	at edu.osu.cse5234.business.view._EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie._invoke(_EJSRemote0SLInventoryServiceBean_78e4cf4b_Tie.java)
	at org.apache.yoko.orb.OBPortableServer.ServantDispatcher.dispatch(ServantDispatcher.java:187)
	at org.apache.yoko.orb.OBPortableServer.POA_impl._OB_dispatch(POA_impl.java:1644)
	at org.apache.yoko.orb.OB.DispatchRequest_impl.invoke(DispatchRequest_impl.java:56)
	at com.ibm.ws.transport.iiop.yoko.ExecutorDispatchStrategy$1.run(ExecutorDispatchStrategy.java:42)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:/Users/derenkong/Desktop/lab6/h2db/MyBookDB.mv.db [1.4.197/7]" [50000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 94 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:/Users/derenkong/Desktop/lab6/h2db/MyBookDB.mv.db [1.4.197/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:870)
	at org.h2.mvstore.FileStore.open(FileStore.java:173)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:350)
	... 88 more
2018-10-24 16:05:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT *FROM CUSTOMER_ORDER BY CUSTOMER_NAME[*]"; SQL statement:
SELECT *FROM CUSTOMER_ORDER BY CUSTOMER_NAME [42000-197]
2018-10-24 16:05:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT *FROM CUSTOMER_ORDER BY CUSTOMER_NAME[*]"; SQL statement:
SELECT *FROM CUSTOMER_ORDER BY CUSTOMER_NAME [42000-197]
2018-10-24 16:06:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT *FROM CUSTOMER_ORDER BY CUSTOMER_NAME[*]"; SQL statement:
SELECT *FROM CUSTOMER_ORDER BY CUSTOMER_NAME [42000-197]
2018-10-24 16:06:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * CUSTOMER_ORDER[*] ORDER BY CUSTOMER_NAME"; SQL statement:
SELECT * CUSTOMER_ORDER ORDER BY CUSTOMER_NAME [42000-197]
2018-10-24 16:06:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * CUSTOMER_ORDER[*] ORDER BY CUSTOMER_NAME"; SQL statement:
SELECT * CUSTOMER_ORDER ORDER BY CUSTOMER_NAME [42000-197]
